packages/components/src/components/page-error/page-error.jsx

```diff
@@ -25,16 +25,16 @@ const PageError = ({
 
     return (
         // if image_url is passed we should split the page to two columns and left-align messages
-        <div className={classNames('dc-page-error__container', { 'dc-page-error__container--left': !!image_url })}>
+        <div className={classNames('dc-page-error__container')}>
             {!!image_url && (
                 <DesktopWrapper>
                     <img
                         className={classNameImage}
                         src={image_url}
                         alt={'404'}
                         loading='lazy'
-                        width='771px' // width and height should be specified so it doesn't jump to the right after image loads
-                        height='448px'
+                        width='607px' // width and height should be specified so it doesn't jump to the right after image loads
+                        height='366px'
```

Comment:
Could we use `rem` instead of `px`?

Reply:
I think we can still use `px` if we want to be precise for larger values.

Reply:
I think it's better to use pixels for this picture, as with `rem` units, the actual picture size and dimensions might change depending on users setting which we don't want. Also, this picture doesn't need to be responsive as it is not shown on smaller screens in general.